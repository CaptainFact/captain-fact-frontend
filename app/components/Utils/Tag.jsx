import React from 'react'
import classNames from 'classnames'


const Tag = ({ type, size = 'small', className, children, ...props }) => (
  <div
    className={classNames('tag', `is-${size}`, (type ? `is-${type}` : null), className)}
    {...props}
  >
    { children }
  </div>
)

export default Tag
