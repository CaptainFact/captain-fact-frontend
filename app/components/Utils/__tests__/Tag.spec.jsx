import Tag from '../Tag'

test("render tag", () => {
  snapshot(<Tag>Test</Tag>)
})

test("set size", () => {
  snapshot(<Tag type="info">Test</Tag>)
})
