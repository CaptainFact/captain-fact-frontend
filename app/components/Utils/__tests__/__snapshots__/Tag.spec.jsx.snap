// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tag 1`] = `
<div
  className="tag is-small"
>
  Test
</div>
`;

exports[`set size 1`] = `
<div
  className="tag is-small is-info"
>
  Test
</div>
`;
